pkgs/development/python-modules/thorlabspm100/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+, pytest-runner
+, setuptools-scm
+}:
+
+buildPythonPackage rec {
+  version = "1.2.2";
+  pname = "ThorlabsPM100";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "881cec8149b17c77d4af0097ee3fdb40eaeb618c876d49f1bc42c87611737967";
```

```suggestion
    hash = "";
```
Would you please use SRI hash here?

---

You can get the SRI hash with the following command.
`nix hash to-sri --type sha256 881cec8149b17c77d4af0097ee3fdb40eaeb618c876d49f1bc42c87611737967`